liquidprompt

```diff
@@ -2343,6 +2348,28 @@ _lp_runtime_color() {
     lp_runtime_color="${LP_COLOR_RUNTIME}${lp_runtime_format}${NO_COL}"
 }
 
+######################
+# Kubernetes context #
+######################
+
+_lp_kube_context() {
+   (( LP_ENABLE_KUBECONTEXT )) || return
+
+   local _LP_KUBE_CONTEXT=$(kubectl config current-context 2>/dev/null)
```

Comment:
Make this local lowercase.

Is the last line's check needed? Could it be moved earlier to save time? The docs don't say if it errors if there is no context: https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#config
But in my testing it errors:
> error: current-context is not set

```suggestion
   local kubernetes_context
   kubernetes_context=$(kubectl config current-context 2>/dev/null) || return 1
```